index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+  <link rel="stylesseet" href="styles.css" >
```

Comment:
🤪 It seems that due to a small error, your Html file is not linking with your styles in the CSS file, it's nothing serious, just correct the link.
```suggestion
  <link rel="stylesheet" href="styles.css" >
```